client/my-sites/upgrades/domain-management/edit-contact-info/form-card.jsx

```diff
@@ -167,21 +189,22 @@ class EditContactInfoFormCard extends React.Component {
 					isPrimary: true
 				}
 			],
-			oldEmail = this.props.contactInformation.email,
-			newEmail = formState.getFieldValue( this.state.form, 'email' );
+			currentEmails = this.getCurrentEmails();
 
 		let text;
-		if ( oldEmail === newEmail ) {
+		if ( this.hasEmailChanged() ) {
 			text = translate( 'To finish this process, this change will need to be confirmed by an email ' +
 				'sent to {{strong}}%(email)s{{/strong}}. Please make sure you have access to it.', {
-					args: { email: newEmail }, components: { strong }
+					args: { email: currentEmails }, components: { strong }
 				}
 			);
 		} else {
+			const newEmail = formState.getFieldValue( this.state.form, 'email' );
+
 			text = translate( 'To finish this process, this change will need to be confirmed by emails ' +
```

Comment:
Hi! I've found a possible matching string that has already been translated **17** times:
`translate( 'To finish this process, this change will need to be confirmed by emails sent to {{strong}}%(oldEmail)s{{/strong}} and {{strong}}%(newEmail)s{{/strong}}. Please make sure you'll be able to do so.' )` *ES Score: 13*
See 1 additional suggestion in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/third-email-whois-update)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).